sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/Models/ResourceType.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Reservations.Models
+{
+    /// <summary> The resource types. </summary>
+    public readonly partial struct ResourceType : IEquatable<ResourceType>
```

Comment:
I think we'd better avoid using `ResourceType` as we already has a definition in `Azure.Core`.

Reply:
rename to ResourceTypeName